integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/datamap/TestDataMapCommand.scala

```diff
@@ -204,6 +204,37 @@ class TestDataMapCommand extends QueryTest with BeforeAndAfterAll {
     }
   }
 
+  test("test show datamap: show datamap property related information") {
```

Comment:
add a test case for preagg also, with DM properties and without DMProperties behavior